rebasehelper/helpers/input_helper.py

```diff
@@ -36,6 +35,29 @@ class InputHelper:
 
     """Class for command line interaction with the user."""
 
+    @staticmethod
+    def strtobool(message):
```

Comment:
Shoudn't this be case-insensitive?

Reply:
And perhaps it would be better to make it a nested function of `get_message()`.

Reply:
Good catch with the sensitivity. Not completely sure about making it nested, I feel like it would make it a bit less readable but I see your point that it is most likely not going to be used anywhere but in `get_message`.

Reply:
If you want to leave it as is, I'd at least make `get_message()` a classmethod.